Modix/Modix.csproj

```diff
@@ -8,7 +8,7 @@
     <DefineConstants>TRACE;DEBUG</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Discord.Net" Version="1.0.2" />
+    <PackageReference Include="Discord.Net" Version="2.0.0-beta" />
```

Comment:
Seriously? This PR is big enough without needing to regression test the entire app.

Reply:
An extremely fair point, however, the longer we go without upgrading the more issues we will likely run into that have been fixed in later versions. From what I've been told, 1.0 -> 2.0 is a much less significant jump than 0.9 -> 1.0 was, so I don't anticipate any major bugs (and there were very few breaking API changes, which I have addressed). 

However, it's not required for the rest of the features in the PR, so I'm happy to revert it all the same if you think it's too risky.

Reply:
Aaahhhhh, is that where all the `embed.Build()` stuff came from?

At this point, if you say you've done the dilligence, I'll take your word for it. I was kinda led to believe that 1.0 => 2.0 was a pretty huge leap. I must have picked up on people complaining about 0.9 => 1.0 and misinterpreted.

I will ask to hold off on merging this until I've had a chance to do some smoke testing of Moderation, and the other features I'm intimate with.